recipes/easydev/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version = "0.9.32" %}
+{% set name = "easydev" %}
+{% set sha256 = "edcaca76933ff0e6adc5e42a914d776bffb8e32d2405622d6c369372d6a6c874" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/23/19/7bf8e1a8dacd4844a2425a1b906150b496e59541bacb24901600501f8ca6/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - easydev_buildPackage=easydev.package:buildPackage
+    - multigit=easydev.multigit:main_func
+    - browse=easydev.browser:main
+    - ibrowse=easydev.browser:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+
+  run:
+    - python
+    - setuptools
+    - colorama
+    - pexpect
+
+test:
+  imports:
+    - easydev
+    - easydev.progressbar
+
+about:
+  home: 'http://packages.python.org/easydev/'
+  license: BSD
```

Comment:
What version of BSD (e.g. BSD 3-Clause)?

Can also add `license_family: BSD` if you wish a more general field like this one.